KarhooUISDK/Screens/QuoteList/QuoteListPresenter.swift

```diff
@@ -158,21 +153,19 @@ final class KarhooQuoteListPresenter: QuoteListPresenter {
         }
         
         let noQuotesInSelectedCategory = quotesToShow.isEmpty && fetchedQuotes.all.isEmpty == false
-        let noQuotesForSelectedParapeters = quotesToShow.isEmpty && fetchedQuotes.all.isEmpty && fetchedQuotes.status == .completed
+        let noQuotesForTimeAndArea = fetchedQuotes.all.isEmpty // && fetchedQuotes.status == .completed
```

Comment:
Why  && .completed condition removed?

Reply:
Because the BE request contains information about area and time. If there are no results in response, we do not need to check if a request is done.